spec/ReviewsSummary.test.js

```diff
@@ -5,10 +5,13 @@ import 'jest-enzyme';
 import 'jest-styled-components';
 import { mount } from 'enzyme';
 
-import { reviews } from '../__mocks__/mockData.js';
-import ReviewsSummary from '../client/src/components/ReviewsSummary.jsx';
+import { reviews } from '../__mocks__/mockData.js'; //eslint-disable-line
+import ReviewsSummary from '../client/src/components/ReviewsSummary.jsx'; //eslint-disable-line
 
 describe('ReviewsSummary component', () => {
+  // test('pause', () => {
```

Comment:
Might be a good idea to remove this commented code for clarity. But I wouldn't worry about it until later though.